lesson02/assignment/index.html

```diff
@@ -8,8 +8,154 @@
         <title>Assignment 02</title>
     </head>
     <body>
+        
+
         <main class="main">
-            <h1>title</h1>
+            <div class="container">
+                <div class ="flex-box1">
+                    <div class ="flex-box2"> 
+                        <div class="static-block">
+                            <img class="round" src="img/headshot.jpg" alt=" "  height="200em">
+                            <h1 class="static-block__title">Paolo Maldini</h1>
+                            <h2 class="slant-text">Front End Developer</h2>
+                
+                        </div>
+                        </div>
+                     
+                            <div class ="flex-box3">
```

Comment:
Per office hours, I often try to keep labels and data near each other in the markup unless there is a specific layout reason not to.  Instead of two mini containers, you could use one and have the label and data as sibling elements in a flex container for alignment.